routes/topics.js

```diff
@@ -2,17 +2,26 @@ const express = require('express');
 const router = express.Router();
 const suggestion = require('../module/suggestion');
 
-router.get('/single/:topicName', async (req, res) => {
-	const verbose = req.query.verbose ? req.query.verbose === 'true' : true;
-	const result = await suggestion.topics(req.params.topicName, verbose);
-	res.status(200).json(result);
-	return;
-});
+const supportedClusterSelections = ['3', '5', '10', '20'];
+
+const defaultClusterSelection = '20';
```

Comment:
Would it help to have a `defaults` object and subsequently calling `defaults.verbosity`, `defaults.type`, etc.?

Reply:
Updated :)